### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by blogmangit** [florisboard/florisboard#1386] [Q&A]
Almost 8 days but no new release in beta thats something unexpected.  Next word suggestions will make the keyboard complete

**Comment by patrickgold** [comment]
Not really, my beta releases _should_ be weekly but if it is not possible to do proper releases within a week it will take another few days for the next beta release. The current debug builds still have some bugs with the localization preferences being reset every time the preference UI is opened and I can't ship such state to the beta branch, so the beta release is delayed. Hope you understand

**Reply by blogmangit** [reply]
Yes man I understand totally 👍👍
